src/screens/app/Components/UserCaption/UserCaption.js

```diff
@@ -1,23 +1,32 @@
 import React from 'react';
 import {Subheading} from 'react-native-paper';
 import {Text, StyleSheet} from 'react-native';
+import PropTypes from 'prop-types';
 
 export default class UserCaption extends React.Component {
   constructor(props) {
     super(props);
-  }
 
-  handleCommentClick = () => {};
+    this.state = {
+      caption: this.props.caption,
+      username: this.props.username,
+    };
+  }
 
   render() {
     return (
       <Subheading style={styles.SubheadingStyle}>
-        <Text style={styles.boldStyle}>{this.props.username}</Text>{' '}
-        <Text style={styles.captionStyles}>{this.props.caption}</Text>
+        <Text style={styles.boldStyle}>{this.state.username}</Text>{' '}
+        <Text style={styles.captionStyles}>{this.state.caption}</Text>
       </Subheading>
     );
   }
 }
+UserCaption.propTypes = {
+  caption: PropTypes.string.isRequired,
```

Comment:
By isRequired, are we only focusing on it being a string? or a non empty string? Because if no caption is provided, of course we just leave it as the empty string ''

Reply:
yea I will be adding empty string as a test, it should work